src/test/regress/expected/foreign_key_restriction_enforcement.out

```diff
@@ -249,6 +420,35 @@ DETAIL:  Reference relation "reference_table" is modified, which might lead to d
      1
 (1 row)
 
+ROLLBACK;
+BEGIN;
+	UPDATE transitive_reference_table SET id = 101 WHERE id = 99;
+DEBUG:  switching to sequential query execution mode
+DETAIL:  Reference relation "transitive_reference_table" is modified, which might lead to data inconsistencies or distributed deadlocks via parallel accesses to hash distributed relations due to foreign keys. Any parallel modification to those hash distributed relations in the same transaction can only be executed in sequential query execution mode
+	SELECT count(*) FROM on_update_fkey_table WHERE value_1 = 101 AND id = 99;
+ count 
+-------
+     0
```

Comment:
Maybe we need to create the tables in such a way that changes to `transitive_reference_table` cascades to `on_update_fkey_table` ...